infra/helper.py

```diff
@@ -212,6 +215,7 @@ def get_parser():  # pylint: disable=too-many-statements
   generate_parser = subparsers.add_parser(
       'generate', help='Generate files for new project.')
   generate_parser.add_argument('project')
+  generate_parser.add_argument('language')
```

Comment:
can we make this a `--language` arg instead? We can also set choices=['cxx', ...]

Reply:
+1

Reply:
done